### PR TITLE
Minor changes to wolfSSL MQTT demo files

### PR DESCRIPTION
**Pull request by lundinc2**
Minor changes to wolfSSL MQTT demo files

Description
-----------
Clean up demo config for wolfSSL mutual auth. 
Add comment about picking the correct ROOT CA.
uncrustify using_wolfSSL.c

Test Steps
-----------

Related Issue
-----------

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: FreeRTOS-Plus/Source/Application-Protocols/network_transport/freertos_plus_tcp/using_wolfSSL/using_wolfSSL.c [hunk 1]
```diff
@@ -382,21 +394,22 @@ TlsTransportStatus_t TLS_FreeRTOS_Connect( NetworkContext_t * pNetworkContext,
...
     pNetworkContext->sslContext.ssl = NULL;
 
     /* Call socket shutdown function to close connection. */
-    Sockets_Disconnect(pNetworkContext->tcpSocket);
+    Sockets_Disconnect( pNetworkContext->tcpSocket );
 
     /* free WOLFSSL_CTX object*/
-    WOLFSSL_CTX* pCtx = pNetworkContext->sslContext.ctx;
-    wolfSSL_CTX_free(pCtx);
+    WOLFSSL_CTX * pCtx = pNetworkContext->sslContext.ctx;
```
Comment (aggarw13) [inline FreeRTOS-Plus/Source/Application-Protocols/network_transport/freertos_plus_tcp/using_wolfSSL/using_wolfSSL.c:410]: (Nit) To be consistent with our coding style, the `pCtx` declaration can be at the top of the function body.

File: FreeRTOS-Plus/Source/Application-Protocols/network_transport/freertos_plus_tcp/using_wolfSSL/using_wolfSSL.c [hunk 2]
```diff
@@ -86,214 +86,226 @@ static TlsTransportStatus_t tlsSetup( NetworkContext_t * pNetworkContext,
...
     TlsTransportStatus_t returnStatus = TLS_TRANSPORT_SUCCESS;
- 
-    /*char errString[80];*/
 
-    configASSERT(pNetCtx != NULL);
-    configASSERT(pHostName != NULL);
-    configASSERT(pNetCred != NULL);
-    configASSERT(pNetCred->pRootCa != NULL);
-    configASSERT(pNetCtx->tcpSocket != NULL);
+    /*char errString[80];*/
```
Comment (aggarw13) [inline FreeRTOS-Plus/Source/Application-Protocols/network_transport/freertos_plus_tcp/using_wolfSSL/using_wolfSSL.c:202]:
```suggestion
```

File: FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Mutual_Auth_wolfSSL/demo_config.h [hunk 3]
```diff
@@ -111,16 +111,19 @@ extern void vLoggingPrintf(const char* pcFormatString,
...
  *
- * Must include the PEM header and footer:
- * "-----BEGIN CERTIFICATE-----\n"\
- * "...base64 data...\n"\
- * "-----END CERTIFICATE-----\n"
+ * @note This certificate should be PEM-encoded.
+ * @note define the path to the certificate, do not copy and paste the certificate
+ * contents to this macro.
+ * @warning If wolfSSL cannot verify the peer when connecting to AWS IoT, try
+ * using the root CA found at
```
Comment (aggarw13) [inline FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Mutual_Auth_wolfSSL/demo_config.h:119]: (Nit) May be good to mention the name of the cross-signing root CA provider.
```suggestion
 * using the root CA of Starfield Services found at
```

File: FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Mutual_Auth_wolfSSL/demo_config.h [hunk 4]
```diff
@@ -111,16 +111,19 @@ extern void vLoggingPrintf(const char* pcFormatString,
...
  * https://docs.aws.amazon.com/iot/latest/developerguide/server-authentication.html#server-authentication-certs
  *
- * @note This certificate should be PEM-encoded.
  *
- * Must include the PEM header and footer:
- * "-----BEGIN CERTIFICATE-----\n"\
- * "...base64 data...\n"\
- * "-----END CERTIFICATE-----\n"
+ * @note This certificate should be PEM-encoded.
+ * @note define the path to the certificate, do not copy and paste the certificate
```
Comment (aggarw13) [inline FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Mutual_Auth_wolfSSL/demo_config.h:116]:
```suggestion
 * @note Define the path to the certificate, do not copy and paste the certificate
```

File: FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Mutual_Auth_wolfSSL/demo_config.h [hunk 5]
```diff
@@ -129,15 +132,12 @@ extern void vLoggingPrintf(const char* pcFormatString,
  * https://docs.aws.amazon.com/iot/latest/developerguide/client-authentication.html
  *
  * @note This certificate should be PEM-encoded.
- *
- * Must include the PEM header and footer:
- * "-----BEGIN CERTIFICATE-----\n"\
- * "...base64 data...\n"\
- * "-----END CERTIFICATE-----\n"
+ * @note define the path to the certificate, do not copy and paste the certificate
```
Comment (aggarw13) [inline FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Mutual_Auth_wolfSSL/demo_config.h:135]:
```suggestion
 * @note Define the path to the certificate, do not copy and paste the certificate
```

**Review by aggarw13 — APPROVED** [review]

**Review by n9wxu — APPROVED** [review]

**Pull request merged**